docs/en/ingest-management/fleet/api-generated/template/index.mustache

```diff
@@ -0,0 +1,170 @@
+////
+This content is generated from the open API specification.
+Any modifications made to this file will be overwritten.
+////
+
+++++
+<div class="openapi">
+  <h2>Access</h2>
+  {{#hasAuthMethods}}
+    <ol>
+    {{#authMethods}}
+      <li>{{#isBasic}}HTTP Basic Authentication{{/isBasic}}{{#isOAuth}}OAuth AuthorizationUrl:{{authorizationUrl}}TokenUrl:{{tokenUrl}}{{/isOAuth}}{{#isApiKey}}APIKey KeyParamName:{{keyParamName}} KeyInQuery:{{isKeyInQuery}} KeyInHeader:{{isKeyInHeader}}{{/isApiKey}}</li>
+    {{/authMethods}}
+    </ol>
+  {{/hasAuthMethods}}
+
+  <h2><a name="__Methods">Methods</a></h2>
+  [ Jump to <a href="#__Models">Models</a> ]
+
+  {{! for the tables of content, I cheat and don't use CSS styles.... }}
+  <h3>Table of Contents </h3>
+  <div class="method-summary">{{access}}</div>
+  {{#apiInfo}}
+  {{#apis}}
+  {{#operations}}
+  <h4><a href="#{{baseName}}">{{baseName}}</a></h4>
```

Comment:
Is this `baseName` variable where this `Default` string in the preview comes from? 

![image](https://user-images.githubusercontent.com/6766512/221650523-03004d0c-ce09-41b0-ba39-b3cea61a151c.png)

If so, what controls this? Can the OpenAPI spec Fleet provides define something more meaningful that'll get rendered here?

Reply:
Similar to the above, it looks like we can just update the index.mustache file directly to adjust most of the formatting. I removed this text. 

Note: I originally changed the text to something meaningful, but after adding tags to control the output order, this heading appeared above every endpoint, so I thought it better to just chop it out.